camera_utils.cpp

```diff
@@ -0,0 +1,77 @@
+#include "camera_utils.hpp"
+
+#include <interfaces/image_format.hpp>
+
+#include <iostream>
+
+namespace bnb
+{
+
+    void camera_utils::full_image_to_pixel_buffer(bnb::full_image_t &image, pixel_buffer_callback_t callback)
```

Comment:
this function just a converter, why it is asynchronous (accepts some callback as parameter)?
it is completely not required there...